src/status_im2/contexts/onboarding/identifiers/style.cljs

```diff
@@ -1,7 +1,8 @@
 (ns status-im2.contexts.onboarding.identifiers.style)
 
 (def page-container
-  {:flex 1})
+  {:flex     1
+   :overflow :hidden})
```

Comment:
Fix for a glitch when navigating to other screen and the carousel overflows the screen bounds